frontend/src/assets/css/variables.scss

```diff
@@ -0,0 +1,6 @@
+$table-background-color: #242636;
```

Comment:
use css custom properties intestead
use the same names for variables from figma